src/util/fees.cpp

```diff
@@ -26,16 +30,35 @@ std::string StringForFeeReason(FeeReason reason) {
     return reason_string->second;
 }
 
-bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
-    static const std::map<std::string, FeeEstimateMode> fee_modes = {
+const std::vector<std::pair<std::string, FeeEstimateMode>>& FeeModeMap() {
+    static const std::vector<std::pair<std::string, FeeEstimateMode>> fee_modes = {
         {"UNSET", FeeEstimateMode::UNSET},
         {"ECONOMICAL", FeeEstimateMode::ECONOMICAL},
         {"CONSERVATIVE", FeeEstimateMode::CONSERVATIVE},
+        {(CURRENCY_UNIT + "/KB"), FeeEstimateMode::BTC_KB},
+        {(ToUpper(CURRENCY_ATOM) + "/B"), FeeEstimateMode::SAT_B},
     };
-    auto mode = fee_modes.find(mode_string);
+    return fee_modes;
+}
 
-    if (mode == fee_modes.end()) return false;
+std::string FeeModes(const std::string& delimiter = ", ") {
```

Comment:
Sorry to keep nit-picking this, but you set a delimiter every time this function is called. But then you set a default value in the header, as well as in the cpp file. At the very least the default should be removed from the cpp file. I'd argue to remove it in all places.

Reply:
A default `", "` seemed like a sensible thing, but you're right that I don't use the default anywhere.

Also, huh. I thought re-declaring the defaults in the .cpp file would result in a compiler error. Regardless, removed both.